Scripts/Utilities/AssetDropUtils.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿#if UNITY_EDITOR
```

Comment:
Probably not in this PR, but we should switch to asmdef files for this purpose

Reply:
i agree completely.  and yeah, we should do it in a PR that does that and only that for the whole project.